### DISCUSSION TITLE
8k onnx vad model

### DISCUSSION THREAD

**Discussion by cosmo1995** [snakers4/silero-vad#149] [Q&A] [answered]
Do you have plans to release the 8k onnx vad model?

**Comment by snakers4** [comment, marked as answer]
We decided to have only "the one" model from now now to avoid complexity and for easier support.

Though we could port a model with an if statement to ONNX (but ifs there are ugly), we could not do so via scripting instead of tracing. We solved several rounds of cryptic erros, but at some point we just gave up.

Probably we will invest some time into it during next major release.

As for now - you can just naively upsample. It will perform worse, but it will work.

**Comment by snakers4** [comment]
https://github.com/snakers4/silero-vad/issues/2#issuecomment-1292317150
